.github/labels.md

```diff
@@ -0,0 +1,249 @@
+and are grouped together by color.  As a rule of thumb there is at least one
+label from every category that needs to be applied to an issue but use common
+sense to check if the label is appropriate or not for the specific Issue.
+
+Currently there are three categories:
+
+
+#### Label category `category`
+
+Labels within this category describe the intent of the Issue.  Usually there
+should be exact one intent for creating an Issue and it should not change during
+the lifetime of an Issue.
+
+#### Label category `component`
+
+The component lists the parts of the project this Issue affects or is affected
+by.  This should help developers to identify if they have required knowledge or
+interests to handle the issue.  There should be at least one component listed
+for an Issue.
+
+#### Label category `status`
+
+With the status category developers, managers or issue reporters can check if
+an issue needs further interaction and who should carry out this interaction.
+Issue have exactly one status.  Status may change over the lifetime of the
+Issue.
+
+
+## Labels
+
+This section lists when an label should be applied and what additional actions
+may be taken.
+
+
+#### Label `category:bug`
+
+The Issue describes or solves a perceived malfunction within the game.
+
+#### Label `category:feature`
+
+The Pull Request contains a new game feature.  Features are parts of the game
+which a player can interact with e.g.: A new species, a new gameplay mechanic
+or a new command line switch for the server that changes gameplay behavior.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Pull Request contains a new or improved implementation of an existing
+feature or any other code restructuring that doesn't modify the current game
+mechanics or representation.
+
+#### Label `category:request`
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The Pull Request contains insignificant code changes that don't involve
+features, refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue deals with the python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue deals with creating game assets or verification if the assets match
+the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue deals with CMake, Visual Studio, Xcode or the general build process.
+
+#### Label `component:common`
+
+??? No idea.
+
+#### Label `component:deployment`
+
+The Issue deals with the deployment process or the software used create the
+installer.
+
+#### Label `component:documentation`
+
+The Issue deals with Doxygen or primarily contains documentation.
```

Comment:
Yes, that distinction makes sense. Maybe change the labels to "component:technical documentation" and "component:user manuals" to better convey the intended purpose?

And if we decide to use this distinction, then we need to clarify in the description of these labels that this is not about "external" vs. "internal" (as: in-game pedia) documentation, but about documentation about the game mechanics vs. documentation about game development.

Which means that e.g. issues requesting a certain change/addition to the quick play wiki page would also fall under "component:user manual" (of course).

> Offtopic: did you create this comment with 'Add single comment' or did you press 'Start review' before creating this comment?

The latter. Well, to be precise, I pressed "Start review" a couple of comments before, this one wasn't the first one which started the review. And before I started the review, I made some single comments in reply to other comments by others above.